pipelines/extension-build.yml

```diff
@@ -0,0 +1,43 @@
+pool:
+  vmImage: 'ubuntu-latest'
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- task: PowerShell@2
+  displayName: Update version in the tasks patch using BuildId 
+  inputs:
+    filePath: 'pipelines/scripts/bump_version.ps1'
+    arguments: '$(Build.BuildId)'
+    pwsh: true
+
+- script: |
+    npm install
```

Comment:
Any compelling reason to use `script` task instead of `Npm@1` task? You may not need the `NodeTool` piece above...